.pre-commit-config.yaml

```diff
@@ -14,7 +14,7 @@ repos:
   - id: flake8
 
 - repo: https://github.com/pre-commit/mirrors-mypy
-  rev: 'v0.982'
+  rev: 'v1.0.1'
```

Comment:
Whenever we bump the mypy version, it tends to require a ton of small fixes across the package. Because of that, it would be better to do this separately from the ParamSpec thing I think.

Reply:
Got it, makes sense.